example/basic/example.dart

```diff
@@ -21,10 +21,13 @@ main() {
         name: 'one',
         defaultRoute: true,
         path: '/one',
-        pageTitle: 'Route One',
+        pageTitle: (_) => 'Route One',
```

Comment:
I'm usually against it.. but I think it might make sense in this case. I imagine there will be a lot of routes with static titles